[{"exception": "org.eclipse.core.runtime.CoreException", "inner": [], "outer_pre": [], "outer_suf": [], "all": []}, {"exception": "exception", "inner": [{"provider": {"package": "org.eclipse.update.internal.configurator", "class": "Utils", "method": "newCoreException", "filename": "Utils", "line": "67"}, "caller": {"package": "org.eclipse.osgi.framework.eventmgr", "class": "EventThread", "method": "run", "filename": "EventThread", "line": "104"}}], "outer_pre": [], "outer_suf": [{"provider": {"package": "org.eclipse.update.internal.configurator", "class": "Utils", "method": "newCoreException", "filename": "Utils", "line": "67"}, "caller": {"package": "org.eclipse.osgi.framework.eventmgr", "class": "EventThread", "method": "run", "filename": "EventThread", "line": "104"}}], "all": [{"provider": {"package": "org.eclipse.update.internal.configurator", "class": "Utils", "method": "newCoreException", "filename": "Utils", "line": "67"}, "caller": {"package": "org.eclipse.osgi.framework.eventmgr", "class": "EventThread", "method": "run", "filename": "EventThread", "line": "104"}}]}]